.github/workflows/lib-codeql.yaml

```diff
@@ -19,7 +19,7 @@ jobs:
     steps:
     - name: Checkout repository
       uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4
-    - uses: actions/setup-go@0a12ed9d6a96ab950c8f026ed9f722fe0da7ef32 # v5
+    - uses: actions/setup-go@b26d40294f8ad76fcc90b915dac85892322fe62d # v5
```

Comment:
@mythi is the `0a12` required or can it be bumped to newer? I guess changing it to `v5` would be even better then tagging actual sha (unless specific version is required)

Reply:
`0a15` is the tag for 5.0.2. `b26d` seems like a glitch since there's no release for it in the repo yet.